### ISSUE TITLE
browser.visit does not exist?

### ISSUE THREAD

**Issue by xinghul** [assaf/zombie#894]
Hi Assaf,

I recently updated to 4.0.7 from 2.5.x, I copy and paste the sample code, it doesn't work:

TypeError: undefined is not a function

Which points to `browser.visit()`.

Here's what I did:

``` javascript

var Browser = require('zombie');
Browser.localhost('example.com', 3000);

describe('...', function() {

  var browser = new Browser();

  describe('...', function() {

    before(function() {
      return browser.visit('/');
    });

```

Any ideas? Thanks

**Comment by assaf** [comment]
Here's one of the tests that would fail if `browser.visit` didn't exist: https://github.com/assaf/zombie/blob/master/test/browser_object_test.js#L43

**Comment by xiaoshao** [comment]
try var Browser = require('zombie').Browser; browser = new Browser();  I think it will works.

**Comment by ghu21st** [comment]
Using the latest Zombie 4.0.7 with nodejs 0.12.2, I found the same issue 'TypeError: undefined is not a function' on browser.visit(). 

However, if I uninstalled Zombie 4.0.7 and installed back to earlier version 3.1.1, then it works! 

It seems after Zombie upgraded to version 4.x it has compatible issue with nodejs 0.12.x. 

Hopefully the next Zombie 4.x version can fix this issue as we do need to keep zombie updated to get the  new changes and latest fixes...
